src/Fixer/Operator/AlignmentFixerHelper.php

```diff
@@ -107,6 +113,28 @@ public function fixAlignment(Tokens $tokens, array $toAlign)
     }
 
     /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return false|int
+     */
+    private function isEqualPartOfDeclareStatement(Tokens $tokens, $index)
```

Comment:
duplicate of the one in the fixer, not sure about the composition we want though...